gamemaker.py

```diff
@@ -1,12 +1,24 @@
 class GameMaker:
   def __init__(self, database):
     self.database = database
-    self.loggedin = false
   
   def login(self, name, password):
-    if (self.database.gameMakerCred = {name, password}):
-      self.loggedin = true
-  //TODO Implement checkStatus
+    returnValue = ""
+    if (self.database.curUser != name && self.database.gameMakerCred = {name, password}):
```

Comment:
- Remove parentheses after if
- curUser is of type User, and name is a string; change this to self.database.curUser.name
- Change && to "and" (without quotes)